pylustrator/QtGuiDrag.py

```diff
@@ -285,7 +284,7 @@ def warnAboutTicks(fig):
                 print("Warning tick and label differ", t, l, "for axes", ax_name, file=sys.stderr)
 
 
-""" Window """
+""" Window. """
```

Comment:
I don't see why a comment that is not a sentence should have a period at the end.